types/office-js-preview/index.d.ts

```diff
@@ -8933,16 +8933,23 @@ declare namespace Office {
          */
         enum OWAView {
             /**
-             * One-column view. Displayed when the screen is narrow. Outlook on the web uses this single-column layout on the entire screen of a smartphone.
+             * Narrow one-column view. Displayed when the screen width from which Outlook on the web is viewed is less than 436 pixels.
```

Comment:
Can we just delete "from which Outlook on the web is viewed"? It makes the syntax of the sentence convoluted and I think it is understood in context (note the "OWA" in the enum name) that we're talking about Outlook on the web.